src/main/java/com/spring/rest/controller/UserController.java

```diff
@@ -48,5 +48,11 @@ public User getUser(@PathVariable String uname) throws UserNotFoundException {
 		
 		return ResponseEntity.status(HttpStatus.CREATED).build();
 	}
+	@RequestMapping(value="/User/{uname}", method = RequestMethod.DELETE)

```

Comment:
Why we are deleting a user by UserName? There can be multiple users with the same name, right?

Reply:
Any update on this?